templates/vue/src/components/Lightbox/media/ActivityMedia/Question/index.vue

```diff
@@ -160,29 +159,8 @@ export default {
         this.question.followUp.nodeId,
         this.question.followUp.questionId
       )
-      let previousAnswers = []
-      if (answerObject !== undefined) {
-        if (this.question.followUp.questionId !== null) {
-          for (const [key, value] of Object.entries(answerObject)) {
-            if (key === "text") {
-              var tempObj = { type: key, answerData: value }
-              previousAnswers.push(tempObj)
-            } else if (key === "dragDrop") {
-              var dragDropObj = { type: key, answerData: value }
-              previousAnswers.push(dragDropObj)
-            } else {
-              var tempAudioObj = {
-                type: key,
-                answerData:
-                  wpData.uploadDirArray.baseurl + "/" + value.url + "?" + Date.now(),
-              }
-              previousAnswers.push(tempAudioObj)
-            }
-          }
-          return previousAnswers
-        }
-      }
-      return []
+
+      return answerObject ? Object.entries(answerObject) : null
```

Comment:
I feel as if this new change will break the audio answer type as this line is needed `answerData:
                  wpData.uploadDirArray.baseurl + "/" + value.url + "?" + Date.now(),
              }`

Reply:
It was moved inside completed-activity-media in order to treat all answers inside one component in the same way.